arrows/mvg/camera_options.cxx

```diff
@@ -151,21 +151,20 @@ void
 camera_options
 ::set_configuration( config_block_sptr config )
 {
-#define GET_VALUE( vtype, vname ) \
+#define VALUE( vtype, vname ) \
```

Comment:
What is the purpose of renaming this?

Reply:
the original name was a bit confusing since we are setting a value by getting it, so I decided to shorten the name to just VALUE

Reply:
Well, it's getting the value from the config (and storing in the instance). Just "value" seems a bit odd; `GET_VALUE_FROM_CONFIG` is maybe "best" but getting awfully long.

Personally, *I* don't have a problem with the old name.

Reply:
well, I read it differently: the function is a setter, as indicated by its name and then it lists the values it is setting

Reply:
back to GET_VALUE by popular demand